### PR TITLE
Add deprecation entry for using the same function as getter and setter

### PR DESCRIPTION
**Pull request by cibernox**

### CODE REVIEW

**Comment by cibernox** [comment]
@rwjblue I'll update the deprecation warning to point to this entry

File: source/deprecations/v1.x.html.md [hunk 1]
```diff
@@ -387,3 +387,37 @@ After:
 
 ```handlebars
 {{#with foo as |bar|}}
+```
+
+#### Deprecate using the same function as getter and setter in Computed Properties
+
+Now that we have the new computed property syntax ([RFC](https://github.com/stefanpenner/rfcs/blob/read-only-cp/0000-read-only-cp-syntax.md))
```
Comment (rwjblue) [inline source/deprecations/v1.x.html.md]: Final location is: https://github.com/emberjs/rfcs/blob/master/active/0001-improved-cp-syntax.md

**Comment by rwjblue** [comment]
One tiny nit-pick about the link to the feature, but other than that it looks good
.

**Comment by cibernox** [comment]
Good catch. Corrected.

**Pull request merged**